src/internal_events/common.rs

```diff
@@ -76,11 +77,9 @@ pub struct StreamClosedError {
 impl InternalEvent for StreamClosedError {
     fn emit(self) {
         error!(
-            message = "Events dropped.",
-            reason = "Downstream is closed.",
+            message = "Failed to forward event(s), downstream is closed.",
             error_code = STREAM_CLOSED,
             error_type = error_type::WRITER_FAILED,
-            intentional = "false",
             stage = error_stage::SENDING,
             count = %self.count,
```

Comment:
Do we still want `count` here? I think no.

Reply:
Good catch :+1: